packages/vscode-ui5-language-assistant/README.md

```diff
@@ -137,6 +137,20 @@ The feature is available in the following:
   - SAPUI5 Namespaces
   - Name of property binding info or aggregation binding info
 
+### XML View Go To Definition
+
+#### Description
+
+Go to definition appears when pressing `Ctr` or `Command` + `hover` on XML attribute's value.
```

Comment:
The tool provides navigation to project files referenced in your xml fragments and views with Go To Definition. The standard keyboard shortcuts are `Ctr` / `Command`+ `Click` and `F12`.